app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "730289765910",
+    "project_id": "tenq-70fac",
+    "storage_bucket": "tenq-70fac.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:730289765910:android:56e9d4938f2abd6e2bff37",
+        "android_client_info": {
+          "package_name": "com.postpc.tenq"
+        }
+      },
+      "oauth_client": [
```

Comment:
What is "oauth"?

Reply:
Some form of an authentication method
https://he.wikipedia.org/wiki/OAuth